src/queueUI/Node.java

```diff
@@ -0,0 +1,6 @@
+package queueUI;
+
+public class Node {
+    int inputData;
+    Node after;
+}
```

Comment:
Please always make the attribues of a class (instance variables) `private`, and if possible `final` (immutable).